src/components/molecules/LoginForm.tsx

```diff
@@ -0,0 +1,65 @@
+import {NavLink} from "react-router-dom";
+import useLoginForm from "../../hooks/useLoginForm";
+import EyeShowPassword from "../icons/EyeShowPassword";
+import AppNextButton from "../atom/AppNextButton";
+
+const LoginForm = () => {
```

Comment:
I personally prefer to declare components that may be end up with some complexity using the class style, this is nice tho.
Some state management would also be nice, so react state could be the only source of truth. But as a starting point, looks really good 👍